examples/violations-ts/store.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 
 const createStore = async () => {
-  const { fetch } = await import("./lib/fetch");
+  const { fetch } = await import("./lib");
```

Comment:
An example with a `../`barrel import would be nice:

```
components
  Button.ts
    import {} from '../sagas' ---> disallowed
sagas
structlintrc --> disallow imports for components './sagas'
```

Reply:
good catch, done. Also a bug with globs like `./components/*` was fixed